PsfLauncher/PSFLauncherMetadata.xml

```diff
@@ -1,5 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<PSFLauncher>
+<PSFData Name="PSFLauncher"
+         xmlns="https://www.w3schools.com"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="https://www.w3schools.com ..\MetadataSchema.xsd">

```

Comment:
I think this will eventually be https://github.com/Microsoft/MSIX-PackageSupportFramework/blob/master/MetadataSchema.xml